packages/carbon-graphs/src/js/helpers/axis.js

```diff
@@ -1205,10 +1225,16 @@ const getAxisLabelWidth = (label, axis, config) => {
  * @param {string} label - Label text
  * @returns {number} label height
  */
-const getAxisLabelHeight = (label) => {
+const getAxisLabelHeight = (label, tickLabelsRotation) => {
   const dummy = d3.select('body').append('div');
   const svg = dummy.append('svg');
   const grouper = svg.append('g');
+
+  if (tickLabelsRotation === -45) {
```

Comment:
To make it future proof for additional values:
```suggestion
  if (tickLabelsRotation !== 0) {
```